### PR TITLE
Error handling unit tests logging

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "discrete-layer-storage",
+  "name": "export-request-storage",
   "version": "1.0.0",
   "description": "discrete layer db service",
```
Comment (asafMasa) [inline package.json:4]: change description

File: src/entity/statuses.ts [hunk 2]
```diff
@@ -27,21 +28,21 @@ export class StatusEntity {
   @Column()
   public progress: number;
 
-  @Column()
-  public status: string;
+  @Column({ type: 'enum', enum: StatusesEnum })
+  public status: StatusesEnum;
```
Comment (asafMasa) [inline src/entity/statuses.ts:32]: 👍

**Review by asafMasa — APPROVED** [review]
🥇

File: package.json (hunk 1 above)
Comment (AnguIar) [reply package.json]: done

**Pull request merged**